lightning/src/offers/static_invoice.rs

```diff
@@ -379,6 +379,7 @@ impl StaticInvoice {
 		self.signature
 	}
 
+	#[allow(unused)] // TODO: remove this once we remove the `async_payments` cfg flag
```

Comment:
looks like that cherry-picking does this, so will fix when the Ci will report all the failure!

Reply:
Fixed in 2354e2c0420b25e941f1c9683046276f35dd7eb7 thanks @valentinewallace